src/encoded/static/components/app.js

```diff
@@ -70,6 +71,50 @@ const biosampleTypeList = [
     'in vitro differentiated cells',
     'induced pluripotent stem cell line'
 ];
+const assayTermNameList = [
```

Comment:
Starting a new instance from a branch off this branch. When drawing the charts, it does a `/search/?type=Experiment&field=@id` and picks through the result to find the assay_title facet, and uses its contents to generate colors.